.github/workflows/update-wordpress-js-dependencies-orchestrator.yml

```diff
@@ -0,0 +1,67 @@
+name: Update WordPress JS Dependencies
+on:
+  workflow_call:
+    inputs:
+      WP_SCRIPT_DIST_TAG:
+        description: The tag to use for updating the dependencies. e.g. wp-6.7
+        required: true
+        type: string
+      PACKAGES:
+        description: Comma separated list of packages to call the update js wordpress dependencies.
```

Comment:
Can we make that description a bit more clear, that:

1. We can define here custom packages, which are not part of the composer.json
2. the format of the package needs to match the "github-organization/repoName"